app/models/concerns/has_role.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 module HasRole
   def role
-    Role.find(role_id)
+    Role.where(id: role_id).first
```

Comment:
roles should never be missing ... but it should behave like other belongs_to ... so this is fine ...